anchors/anchor_confirmation_task.go

```diff
@@ -48,17 +48,11 @@ type anchorConfirmationTask struct {
 	AnchorCommittedFilterer anchorCommittedWatcher
 }
 
-// Name returns AnchorRepositoryConfirmationTaskName
-func (act *anchorConfirmationTask) Name() string {
```

Comment:
IMO, Name seems simple and better. Any reason why we are going with `TaskTypeName`?
It seems too verbose

Reply:
The `Name` function is implemented by many unrelated stuff, I dont want to mix them up.